test/e2e/condition.py

```diff
@@ -0,0 +1,145 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Utility functions to help processing Kubernetes resource conditions"""
+
+# TODO(jaypipes): Move these functions to acktest library. The reason these are
+# here is because the existing k8s.assert_condition_state_message doesn't
+# actually assert anything. It returns true or false and logs messages.
+
+import pytest
```

Comment:
Please make sure (in a future patch) to bump the acktest version and import those utility functions instead

Reply:
Will do :)